beets/autotag/mb.py

```diff
@@ -156,6 +156,11 @@ def track_info(recording, index=None, medium=None, medium_index=None,
     ``medium_index``, the track's index on its medium; ``medium_total``,
     the number of tracks on the medium. Each number is a 1-based index.
     """
+    flag = False
+
+    if recording['title'] == '[silence]':
```

Comment:
Can you please put the magic string `'[silence]'` into a constant at the top of the file? For example, the name `SILENCE_TITLE` would work. (It can go next to the other constants at the top of the file, such as `VARIOUS_ARTISTS_ID`.)